hnn_core/tests/test_cell.py

```diff
@@ -103,6 +104,54 @@ def test_cell():
     assert cell.sections[sec_name].cm == new_cm
     assert cell.sections[sec_name].Ra == new_Ra
 
+    # Testing update end pts using template cell
+    cell1 = pyramidal(cell_name='L5Pyr')
+    cell1.update_end_pts()
+    end_pts_original = list()
+    end_pts_new = list()
+    for sec_name in cell1.sections.keys():
+        section = cell1.sections[sec_name]
+        end_pts_original.append(section.end_pts)
+        section._L = section._L * 2
+        cell1.sections[sec_name] = section
+    cell1.update_end_pts()
+    for sec_name in cell1.sections.keys():
+        section = cell1.sections[sec_name]
+        section._L = section._L / 2
```

Comment:
Why are we dividing by two after calling update endpoints? wouldn't this lead to a mismatch?

Reply:
Definitely think this is the right format for the test, and the explicit checks below are good too, but not sure I totally follow all the logic

Reply:
We divide all section lengths by 2 to get back the original end points because earlier we had doubled them and updated end_pts in line 117.